toolbar-extender/toolbar-items-provider.ts

```diff
@@ -1,10 +1,19 @@
-import { ClassProvider } from "@angular/core";
+import { ClassProvider, Injectable } from "@angular/core";
 import { ToolBarItem, ToolBarItemsProvider, TOOLBARITEMS_TOKEN } from "progress-sitefinity-adminapp-sdk/app/api/v1";
 
 // This is webpack specific loader syntax for injecting css as <style> tag in header
 require("!style-loader!css-loader!./toolbar-items-provider.css");
 
+/**
+ * A custom toolbar provider implementation for counting the words in the html editor.
+ */
+@Injectable()
 class CustomToolBarItemsProvider implements ToolBarItemsProvider {
+
+    /**
+     * The method that gets invoked when the editor constructs the toolbar actions.
+     * @param editorHost The instance of the editor.
```

Comment:
reference to Kendo UI documentation may worth adding

Reply:
in the class header